### PR TITLE
Fixed self-loan component

### LINKED ISSUE #725: AddLoanComponent feature bug
### Description
The v-model connected to the input of AddLoanComponent is bugged and returns [object Object] as default. This further results in articles not being able to be scanned. The OK button seems to not be linked to any type of action.

## 🔬 To Reproduce
Steps to reproduce the behaviour:
-The input field should be empty
-The book should be scanned
-The barcode numbers of the book should appear in the input field
-Clicking OK should result in some type of visual action

### PR DESCRIPTION
**Pull request by ntijoh-evelina-karlsson**
## ⚙️ What issue did you implement or fix?
Fixes #725 

## 📜 Summary
- Fixed style and function for the self loan menu. The code was reworked entirely due to old code being complicated and messy.  

## 📝 Type of change

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 This change requires a documentation update

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by ntijoh-my-lovell — APPROVED** [review]
great work guys

**Review by ntijoh-nikki-andersson — APPROVED** [review]

**Review by ntijoh-Erik-Albinsson — CHANGES_REQUESTED** [review]
According to the projects code standard, 'singe quotes' should be used in frontend

**Comment by ntijoh-evelina-karlsson** [comment]
> According to the projects code standard, 'singe quotes' should be used in frontend

It is not possible to use single quotes in the vuetify tags.

**Review by ntijoh-Erik-Albinsson — APPROVED** [review]

**Comment by Linkaan** [comment]
This PR is 5 commits behind develop, consider merging with develop to make sure all checks pass:
```bash
git merge develop
```

**Review by ntijoh-linus-bjorken — APPROVED** [review]

**Comment by Linkaan** [comment]
Make sure to update develop before merging with
git checkout develop
git pull

Alternatively you could run
git fetch --all
git merge origin/develop

**Review by ntijoh-linus-bjorken — APPROVED** [review]

File: frontend/src/components/AddLoanComponent.vue [hunk 1]
```diff
@@ -5,155 +5,143 @@
...
-            <v-text-field
-              id="pop1"
-              ref="input"
-              v-model="input"
-              :state="inputState"
+            <v-item-group
+              class="mb-1"
+              label="Bok ID"
+              label-for="pop1"
+              horizontal=""
```
Comment (ntijoh-nikki-andersson) [inline frontend/src/components/AddLoanComponent.vue:38]: Empty field?

**Review by ntijoh-david-roman — APPROVED** [review]

**Review by ntijoh-nikki-andersson — APPROVED** [review]

**Pull request merged**